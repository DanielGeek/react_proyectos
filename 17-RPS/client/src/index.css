body, html, div#root {
    background: #f93e17;
    font-family: sans-serif, 'Roboto Condensed';
}
.container {
    margin-top: 40px;
}