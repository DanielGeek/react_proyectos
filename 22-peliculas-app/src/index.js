import React from "react";
import ReactDOM from "react-dom";
import App from "./App";

import "bootstrap/dist/css/bootstrap.css";
import "bootstrap/dist/css/bootstrap-theme.css";

ReactDOM.render(
  <App nombre="Daniel" mensaje="Este es un mensaje" />,
  document.getElementById("root")
);
