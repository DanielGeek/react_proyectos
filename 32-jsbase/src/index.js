const cities = ['Caracas', 'Buenos Aires', 'Santiago', 'Ciudad de MÃ©xico'];

const randomString = () => {
  const strings = cities[Math.floor(Math.random() * cities.length)];
  return strings;
};

const reverseString2 = str => {
  return new Promise((resolve, reject) => {
    if (!str) {
      reject(Error('Error'));
    }
    resolve(str.split("").reverse().join(""));
  });
};

module.exports = randomString;
