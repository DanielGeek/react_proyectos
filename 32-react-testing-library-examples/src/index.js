import React from 'react';
import ReactDOM from 'react-dom';
import { createStore } from 'redux';
import { noteReducer } from './reducers/noteReducer';

const store = createStore(noteReducer);

store.dispatch({
  type: '@notes/created',
  payload: {
    content: 'I like the classes with Dani',
    important: true,
    id: 1
  }
})

store.dispatch({
  type: '@notes/created',
  payload: {
    content: 'But i need learn',
    important: false,
    id: 2
  }
})

const App = () => {
  const state = store.getState();

  return (
    <ul>
      {
        state.map(note => {
          return <li key={note.id}>
            {note.content}
            <strong>
              {
                note.important
                  ? 'important'
                  : 'not important'
              }
            </strong>
          </li>
        })
      }
    </ul>
  )
}

const renderApp = () => {
  ReactDOM.render(
    <App />,
    document.getElementById('root')
  );
}

renderApp()
store.subscribe(renderApp)