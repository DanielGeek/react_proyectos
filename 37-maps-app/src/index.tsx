import React from 'react';
import ReactDOM from 'react-dom';
import { MapsApp } from './MapsApp';

//@ts-ignore
// eslint-disable-next-line import/no-webpack-loader-syntax
import mapboxgl from '!mapbox-gl'; // or "const mapboxgl = require('mapbox-gl');"

mapboxgl.accessToken = 'pk.eyJ1IjoiZGFuaWVsZ2VlayIsImEiOiJja291eHRlODQwMm5hMm5sOXM3YzlwajlxIn0.UORg5svptSrVMXj_FSENAg';

if ( !navigator.geolocation ) {
  alert('You browser not have geolocation option');
  throw new Error('You browser not have geolocation option');
}

ReactDOM.render(
  <React.StrictMode>
    <MapsApp />
  </React.StrictMode>,
  document.getElementById('root')
);

