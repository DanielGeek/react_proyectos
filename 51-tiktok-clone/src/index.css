@font-face {
  font-family: 'ProximaNova';
  src: url('assets/Proxima-Nova-Bold.woff2')
  format('woff2');
  font-weight: 400;
}

@font-face {
  font-family: 'ProximaNova';
  src: url('assets/Proxima-Nova-Semibold.woff2')
  format('woff2');
  font-weight: 600;
}

:root {
  margin: 0;
  font-family: ProximaNova,Arial,Tahoma,PingFangSC,sans-serif;
  font-size: 16px;
  line-height: 24px;
  font-weight: 400;

  color-scheme: light dark;
  color: rgba(255, 255, 255, 0.87);
  background-color: #242424;

  font-synthesis: none;
  text-rendering: optimizeLegibility;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  -webkit-text-size-adjust: 100%;
}

html {
  box-sizing: border-box;
}

*, *::before, *::after {
  box-sizing: inherit;
}


