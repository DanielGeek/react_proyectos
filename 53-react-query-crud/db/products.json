{
  "products": [
    {
      "id": 1,
      "name": "laptop",
      "description": "laptop gaming",
      "price": 1000,
      "inStock": true
    },
    {
      "id": 2,
      "name": "mouse",
      "description": "mouse gaming",
      "price": 1000,
      "inStock": true
    },
    {
      "id": 3,
      "name": "keyboard",
      "description": "keyboard gaming",
      "price": 200,
      "inStock": false
    },
    {
      "name": "server",
      "description": "server wb",
      "price": "1000",
      "inStock": false,
      "id": 4
    }
  ]
}