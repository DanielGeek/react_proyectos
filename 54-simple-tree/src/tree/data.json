{
    "name": "mammals",
    "children": [
        {
            "name": "cheetah",
            "type": "carnivores",
            "children": []
        },
        {
            "name": "bear",
            "type": "carnivores",
            "children": []
        },
        {
            "name": "lion",
            "type": "carnivores-2",
            "children": []
        },
        {
            "name": "dog",
            "type": "carnivores-2",
            "children": []
        },
        {
            "name": "elephant",
            "type": "herbivorous",
            "children": []
        },
        {
            "name": "ape",
            "type": "carnivores",
            "children": []
        }
    ]
}