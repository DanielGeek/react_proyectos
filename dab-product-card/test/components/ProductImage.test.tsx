import React from 'react';
import renderer from 'react-test-renderer';
import { ProductImage, ProductCard } from '../../src/components';
import { product2 } from '../data/products';

describe('ProductImage', () => {

  test('Then should show the component with imagen personalized', () => {

    const wrapper = renderer.create(
      <ProductImage img="https://hola.jpg" />
    )

    expect( wrapper.toJSON() ).toMatchSnapshot();

  });

  test('Then should show the component with the imagen of the product', () => {

    const wrapper = renderer.create(
      <ProductCard product={ product2 }>
        {
          () => (
            <ProductImage />
          )
        }
      </ProductCard>
    )

    expect( wrapper.toJSON() ).toMatchSnapshot();

  });

})