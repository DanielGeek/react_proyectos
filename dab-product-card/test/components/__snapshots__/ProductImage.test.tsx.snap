// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductImage Then should show the component with imagen personalized 1`] = `
<img
  alt="Product"
  className="productImg undefined"
  src="https://hola.jpg"
/>
`;

exports[`ProductImage Then should show the component with the imagen of the product 1`] = `
<div
  className="productCard undefined"
>
  <img
    alt="Product"
    className="productImg undefined"
    src="./coffee-mug.png"
  />
</div>
`;
